src/main/java/seedu/address/model/item/ItemList.java

```diff
@@ -83,4 +83,12 @@ protected ItemList find(String searchString, ItemList il) {
         }
         return il;
     }
+
```

Comment:
Also, maybe you can add a .size() method that returns the length of the list? I feel that this would be good as it allows other classes to check for logic errors :)

Reply:
ok! no problem for that